e2e_playwright/st_set_page_config_test.py

```diff
@@ -108,9 +108,21 @@ def test_page_icon_with_emoji_symbol(app: Page):
     )
 
 
+def test_page_icon_with_local_icon_str(app: Page):
+    click_button(app, "Page Config With Local Icon Str")
+    expect(app).to_have_title("With Local Icon Str")
+    favicon_element = app.locator("link[rel='shortcut icon']")
+    expect(favicon_element).to_have_count(1)
+    expect(favicon_element).to_have_attribute(
+        "href",
+        re.compile(r"d1e92a291d26c1e0cb9b316a93c929b3be15899677ef3bc6e3bf3573\.png"),
```

Comment:
Seems so, since I just copied this from an existing test. But you are right that it's probably better to just check for `.png`, so I changed both tests.